docs/managed-datahub/approval-workflows.md

```diff
@@ -0,0 +1,184 @@
+</p>
+
+3. This proposal will be sent to the inbox of Admins with proposal permissions and data owners.
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/proposed_term_to_dataset_in_inbox.png"/>
+</p>
+
+4. From there, they can choose to either accept or reject the proposal. A full log of all accepted or rejected proposals is kept for each user.
+
+### Proposing additions to your Business Glossary
+
+1. Navigate to your glossary by going to the Govern menu in the top right and selecting Glossary.
+
+2. Click the plus button to create a new Glossary Term. From that menu, select Propose.
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/proposing_new_glossary_term.png"/>
+</p>
+
+3. This proposal will be sent to the inbox of Admins with proposal permissions and data owners.
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/inbox_with_new_glossary_proposal.png"/>
+</p>
+
+4. From there, they can choose to either accept or reject the proposal. A full log of all accepted or rejected proposals is kept for each user.
+
+### Proposing Glossary Term Description Updates
+
+1. When updating the description of a Glossary Term, click propse after making your change.
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/propose_glossary_description_change.png"/>
+</p>
+
+2. This proposal will be sent to the inbox of Admins with proposal permissions and data owners.
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/inbox_with_description_proposal.png"/>
+</p>
+
+3. From there, they can choose to either accept or reject the proposal.
+
+## Proposing Programatically
+
+DataHub exposes a GraphQL API for proposing Tags and Glossary Terms.
+
+At a high level, callers of this API will be required to provide the following details:
+
+1. A unique identifier for the target Metadata Entity (URN)
+2. An optional sub-resource identifier which designates a sub-resource to attach the Tag or GLossary Term to. for example reference to a particular "field" within a Dataset.
+3. A unique identifier for the Tag or Glossary Term they wish to propose (URN)
+
+In the following sections, we will describe how to construct each of these items and use the DataHub GraphQL API to submit Tag or Glossary Term proposals.
+
+#### Constructing an Entity Identifier
+
+Inside DataHub, each Metadata Entity is uniquely identifier by a universal resource name, or an URN. This identify can be copied from the entity page, extracted from the API, or read from a downloaded search result. You can also use the helper methods in the datahub python library given a set of components.
```

Comment:
Spelling --  **identified**

Since URN might be a new concept to folks and it's specific to DataHub terminology, let's propcase & bold this:

> Inside DataHub, each Metadata Entity is uniquely identified by a **Universal Resource Name**, or an URN.